src/main/asyncapi/network_survey_messaging.yaml

```diff
@@ -73,6 +73,12 @@ channels:
       message:
         $ref: '#/components/messages/wifiProbeRequestRecord'
 
+  wifi_deauthentication_message:
+    description: The wifi_deauthentication_message topic/channel is where 802.11 (WiFi) Deauthentication Management Frames can be published. For MQTT, set the MQTT topic as "wifi_deauthentication_message". When a station wants to disassociate from another station, it invokes the deauthentication service. Deauthentication is a notification and cannot be refused. A station performs deauthentication by sending an authentication management frame (or group of frames to multiple stations) to advise of the termination of authentication.
```

Comment:
Could we use the `wifi_ota_message` to hold this record? Instead of having to make a message for each specific 802.11 frame time we can use the generic `wifi_ota_message` to hold all of them.

Reply:
For anyone stumbling on this PR, I wanted to provide an update. We talked offline and are putting this PR on hold for a bit. Once things flush out a bit I will be able to make a better decision on the direction of this PR. I do like the idea of continuing this model for all the wifi messages (placing the wifi records in a typed message), and I will likely go down that road, but I need to investigate a couple areas first.